changelog.d/saf-1001.fixed

```diff
@@ -0,0 +1 @@
+Added support for taint flowing into function declarations on object literals.
```

Comment:
Could you share a small example here? Otherwise I don't think this will be understood by most users. Also, the change helps taint, but doesn't it help const-prop too? It's a dataflow fix more than a taint fix. Same applies to the PR title.

Reply:
E.g. previously Semgrep would not report `sink(t)`, I think:

```javascript
t = "tainted"
...
{
    foo: (x) => {
        sink(t)
    }
}
```